mass_editing/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -7,6 +7,10 @@ def migrate(cr, version):
     if not version:
         return
     # Don't execute if already migrated in v12
```

Comment:
Doesn't this comment correspond to the other statement?

Reply:
This comment is for both